README.md

```diff
@@ -0,0 +1,179 @@
+# periph_board modular board component
+
+*periph_board* is a Viam modular board component that an periph.io based GPIO interface for single board computers.
+
+## Build
+
+On the single board computer that will be running viam-server and the periph_board module, run these commands to install prerequisites and build:
+
+``` bash
+sudo apt update && sudo apt upgrade -y
+sudo apt-get install git
+wget https://go.dev/dl/go1.20.5.linux-arm64.tar.gz
+sudo  rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go1.20.5.linux-arm64.tar.gz
+echo 'PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc && source ~/.bashrc
```

Comment:
Not sure we need the Go install instructions included, especially since they're specific to a version of go, and to linux amd64.